src/components/Settings/SettingsPlex.tsx

```diff
@@ -223,6 +387,62 @@ const SettingsPlex: React.FC<SettingsPlexProps> = ({ onComplete }) => {
                     </div>
                   </div>
                 </div>
+                <div className="mt-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:items-start sm:border-t sm:border-gray-800">
+                  <label
+                    htmlFor="preset"
+                    className="block text-sm font-medium leading-5 text-gray-400 sm:mt-px"
+                  >
+                    <FormattedMessage {...messages.serverpreset} />
+                  </label>
+                  <div className="mt-1 sm:mt-0 sm:col-span-2">
+                    <div className="flex max-w-lg rounded-md shadow-sm input-group">
+                      {/* Disables no-onchange as onBlur does not work as well, no-onchange is depreciated see: */}
+                      {/* https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md */}
+                      {/* eslint-disable-next-line jsx-a11y/no-onchange*/}
```

Comment:
Ah, nice to know this was deprecated. We can diasble the rule in the eslint config then, instead of just here.